MiniGamesBox Classic/src/main/java/plugily/projects/minigamesbox/classic/utils/helper/ArmorHelper.java

```diff
@@ -36,33 +36,33 @@ private ArmorHelper() {
 
   public static void setArmor(Player player, ArmorType type) {
     PlayerInventory inv = player.getInventory();
-    switch(type) {
-      case LEATHER:
+    switch (type) {
+      case LEATHER -> {
```

Comment:
This looks like Java 13 enhanced switch expression feature, which means the plugin won't run nor even compile on 1.12 for example.

Reply:
Resolved, thanks for pointing out.